catboost/python-package/catboost/core.py

```diff
@@ -5074,3 +5074,31 @@ def _plot_feature_statistics(statistics, feature, max_cat_features_on_plot):
     else:
         fig = _build_binarized_feature_statistics_fig(statistics, feature)
         return [fig]
+
+
+def to_regressor(model):
+    regressor = CatBoostRegressor()
```

Comment:
Let's avoid `__init__` call here by this way: `regressor = CatBoostRegressor.__new__(CatBoostRegressor)`